### PR TITLE
Upgrade to go v1.13

### PR DESCRIPTION
**Pull request by chenrui333**
## Description

Upgrade to go v1.13.

Relates to https://github.com/projectcalico/go-build/pull/82, #2061 

Currently, the brew formula is already built with go v1.13, https://github.com/Homebrew/homebrew-core/pull/44132

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
@chenrui333 looks like you've got some test failures.

**Comment by chenrui333** [comment]
Circle back on this. :)

**Comment by chenrui333** [comment]
Should be covered with the latest `go-build` container.

**Comment by chenrui333** [comment]
@caseydavenport it should be good now! Thanks!

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+issues:
+  exclude:
```
Comment (caseydavenport) [inline .golangci.yml]: @chenrui333 could you add a comment explaining why we need to exclude these checks? It's not obvious to me, and I don't want the context to get lost :)

Comment (chenrui333) [reply .golangci.yml]: Sure, I am doing that now.

Comment (chenrui333) [reply .golangci.yml]: Done. :)

Comment (chenrui333) [reply .golangci.yml]: Let me know if you need more things from my side.

**Pull request closed without merging**